.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": ["@babel/plugin-transform-runtime"]
```

Comment:
sorry for being naive but what is this for?

Reply:
`threads.js` - I'll leave a note.